vms/avm/service.md

```diff
@@ -783,6 +783,47 @@ curl -X POST --data '{
 }
 ```
 
+### `info.getTxFee`
```

Comment:
the api docs list the endpoints in alphabetical order. `avm.getTxFee` should come after [avm.getTx](https://docs.avax.network/api-reference/x-chain/api#avmgettx)